package/src/lib/SpotlightTour.provider.tsx

```diff
@@ -217,10 +232,13 @@ export const SpotlightTourProvider = forwardRef<SpotlightTour, SpotlightTourProv
 
   return (
     <SpotlightTourContext.Provider value={tour}>
-      {isChildFunction(children)
-        ? <SpotlightTourContext.Consumer>{children}</SpotlightTourContext.Consumer>
-        : <>{children}</>
-      }
+      {isChildFunction(children) ? (
+        <SpotlightTourContext.Consumer>
+          {children}
+        </SpotlightTourContext.Consumer>
+      ) : (
+        <>{children}</>
+      )}
```

Comment:
I don't see how this is more readable than the way it was before. Now it has unnecessary parenthesis, the `?` and `:` are harder to spot, it's unclear that the expression is a ternary, and it's more difficult to move things around 😞